src/SM2013/SM2012Sph.cpp

```diff
@@ -247,14 +250,14 @@ int SM2012Sph<ndim, kernelclass >::ComputeH
     for (j=0; j<nbody->Nstar; j++) {
       invhsqd = pow(2.0 / (parti.h + nbody->stardata[j].h),2);
       for (k=0; k<ndim; k++) dr[k] = nbody->stardata[j].r[k] - parti.r[k];
-      ssqd = DotProduct(dr,dr,ndim)*invhsqd;
```

Comment:
I removed the ssd variable, so i needed to comment it out. 

I'd rather not delete the SM2012 as it is so close to working. With a little effort I think we can get it working again without the need for code duplication. It's just not a priority at this second...